.circleci/config.yml

```diff
@@ -561,43 +561,45 @@ workflows:
         equal: [ scheduled_pipeline, << pipeline.trigger_source >> ]
     jobs:
       - lint:
-          xcode_version: '13.4.1'
+          xcode_version: '14.0.0'
       - run-test-ios-16:
           xcode_version: '14.0.0'
       - spm-build:
-          xcode_version: '13.4.1'
+          xcode_version: '14.0.0'
       - run-test-ios-15:
-          xcode_version: '13.4.1'
+          xcode_version: '14.0.0'
       - run-test-tvos:
-          xcode_version: '13.4.1'
+          xcode_version: '14.0.0'
       - run-test-ios-14:
-          xcode_version: '13.4.1'
+          xcode_version: '14.0.0'
       - run-test-ios-13:
-          xcode_version: '13.4.1'
+          xcode_version: '14.0.0'
+          <<: *release-branches-and-main
       - run-test-ios-12:
-          xcode_version: '13.4.1'
+          xcode_version: '14.0.0'
+          <<: *release-branches-and-main
       - build-tv-watch-and-macos:
           xcode_version: '13.4.1'
```

Comment:
Should we add a comment here to revisit it in the future?

Reply:
Good idea, added.